src/applications/healthcare/questionnaire/config/form.js

```diff
@@ -30,6 +30,7 @@ const formConfig = {
   formId: VA_FORM_IDS.FORM_HC_QSTNR,
   saveInProgress: {
     messages: {
+      hideInProgressMessage: true,
```

Comment:
This should do the trick now :crossed_fingers: 

```suggestion
      inProgress: '',
```